python/mxnet/contrib/onnx/_import/import_onnx.py

```diff
@@ -126,13 +130,22 @@ def from_onnx(self, graph):
                     len(node.output), len(mxnet_sym.list_outputs()), op_name))
             for k, i in zip(list(node.output), range(len(node.output))):
                 self._nodes[k] = mxnet_sym[i]
+
+            # splitting params into args and aux params
+            for args in mxnet_sym.list_arguments():
+                if args in self._params:
```

Comment:
Why is this if statement required. In what situation will members of mxnet_sym.list_arguments() not be present the self._param list

Reply:
list_arguments also includes input symbols , so for something like conv , list_arguments will look  like:  [ 'input_0' , 'param_0' , 'param_1']  , we need only last two to be added to arg params.
So need to check for params only that exists in param dict.